docs/framework/additional-apis/system.net.connectstream.connection.md

```diff
@@ -0,0 +1,43 @@
+---
+title: ConnectStream.Connection Property (System.Net)
+ms.date: 10/21/2019
+ms.technology: "dotnet-networking"
+topic_type:
+  - "apiref"
+api_name:
+  - "System.Net.ConnectStream.Connection"
+  - "System.Net.ConnectStream.get_Connection"
+api_location:
+  - "System.dll"
+api_type:
+  - "Assembly"
+---
+# ConnectStream.Connection Property
+
+Gets the connection.
+
+## Syntax
+
+```csharp
+internal Connection Connection { get; }
+```
+
+## Property value
+
+`System.Net.Connection`
```

Comment:
I think the intention is to have a line break between this and the next line. so added two spaces.

```suggestion
`System.Net.Connection`  
```